packages/react-native/template/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "react": "18.2.0",
-    "react-native": "1000.0.0"
+    "@callstack/react-native-visionos": "1000.0.0"
```

Comment:
we also need to keep `react-native` in the template to keep `ios` and `android` working in generated project